CheapShopWeb/Controllers/ProductInfoController.cs

```diff
@@ -12,22 +13,18 @@ namespace CheapShopWeb.Controllers
 {
     public class ProductInfoController : Controller
     {
-        private ProductDbContext db;
+        private static List<Product> filtered = new List<Product>();
```

Comment:
Statinis variantas nera gerai siuo atveju, nes sis listas bus vienodas tarp visų vartotojų

Reply:
Tai ten zemiau irgi var'ais tiesiog padaryti reikia arba tiesiai i view'a perduoti rezultata is `await ApiService....`

Reply:
realiai to static net nereikia čia, idk kodėl parašiau ;.)